python/mxnet/ndarray.py

```diff
@@ -1224,6 +1226,12 @@ def add(lhs, rhs):
     >>> (x+y).asnumpy()
     array([[ 1.,  1.,  1.],
            [ 2.,  2.,  2.]], dtype=float32)
+    >>> mx.nd.add(x,y).asnumpy()
+    array([[ 1.,  1.,  1.],
+           [ 2.,  2.,  2.]], dtype=float32)
+    >>> mx.nd.broadcast_add(x,y).asnumpy()
+    array([[ 1.,  1.,  1.],
+           [ 2.,  2.,  2.]], dtype=float32)
```

Comment:
I wanted to show that user can use either of 3 functions and get same output. So, added examples there. I had mentioned the function is equivalent to other two functions in explanation.  Now, I removed the extra examples added.